apps/src/applab/applab.js

```diff
@@ -1453,3 +1453,34 @@ Applab.getAssetDropdown = function (typeFilter) {
   });
   return options;
 };
+
+/**
+ * Return droplet dropdown options representing a list of ids currently present
+ * in the DOM, optionally limiting the result to a certain HTML element tagName.
+ * @param {undefined|string} tagFilter Optional HTML element tagName to filter for.
```

Comment:
I think the right way of doing this is just {string?}, which means optional string (the undefined is then implied as a possible value)